### PR TITLE
Fix tab example

### PR DESCRIPTION
**Pull request by agcolom**

### CODE REVIEW

**Comment by agcolom** [comment]
@arschmitz Hi Alex, this one is fairly straight forward but whoever merges it will have to resolve conflicts.

File: pages/tabs.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 }</script>
 
 <p>This widget is imported unchanged from <a href="http://jqueryui.com/">jQuery UI</a>. Nevertheless, in keeping with jQuery Mobile widget features, the options documented on its jQuery UI API <a href="http://api.jqueryui.com/tabs/">reference</a> page are also available as <code>data-*</code> attributes.</p>
-
+<div class="warning"><b>Note:</b> If you are going to use a UI element on the page in conjunction with the tabs widget, do not use a link, as this will interfere with the management of the <code>ui-btn-active</code> class that is used on the tabs. Use other UI elements, such as <code>&lt;button&gt;</code>, instead.</div>
```
Comment (arschmitz) [inline pages/tabs.html]: Maybe change this to say something more along the lines of  "Use of links to open popup's and panels is not currently recommended in conjunction with the tabs widget. Clicking these links will remove the active state style of the selected tab". This is a very specific issue we don't want to make it sounds bigger then it is

File: pages/tabs.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 }</script>
 
 <p>This widget is imported unchanged from <a href="http://jqueryui.com/">jQuery UI</a>. Nevertheless, in keeping with jQuery Mobile widget features, the options documented on its jQuery UI API <a href="http://api.jqueryui.com/tabs/">reference</a> page are also available as <code>data-*</code> attributes.</p>
```
Comment (arschmitz) [inline pages/tabs.html]: Not directly related but a good time to update. This is no longer true we now override one of the methods to deal with base tag manipulation.

Comment (arschmitz) [inline pages/tabs.html]: Never mind thats only for 1.5

File: pages/tabs.html (hunk 1 above)
Comment (agcolom) [inline pages/tabs.html]: Thanks, Hopefully I'll get a chance to update this today.

**Comment by agcolom** [comment]
@arschmitz updated the warning message according to your suggestion

**Comment by arschmitz** [comment]
:+1:

**Pull request closed without merging**